app/src/components/Slider.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState, useId } from 'react'
+
+const Slider = ({ maxRange }: { maxRange: number }) => {
+  const [sliderValue, setSliderValue] = useState(0)
+  const sliderId = useId()
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const newSliderValue = Number(e.target.value)
+    if (isNaN(newSliderValue)) return
+
+    setSliderValue(newSliderValue)
+  }
+
+  return (
+    <div>
+      <input
+        type="range"
+        name="slider"
```

Comment:
Shouldn't the name be the same as the id?

If we place multiple sliders they will all have the same name :thinking:.